test/JDBC/expected/routines_definition-vu-verify.out

```diff
@@ -1,3 +1,13 @@
+select SPECIFIC_CATALOG, SPECIFIC_SCHEMA, SPECIFIC_NAME, ROUTINE_CATALOG, ROUTINE_SCHEMA, ROUTINE_NAME, ROUTINE_DEFINITION from information_schema.routines where SPECIFIC_NAME LIKE 'xp_%' ORDER BY ROUTINE_NAME;
+go
+~~START~~
+nvarchar#!#nvarchar#!#nvarchar#!#nvarchar#!#nvarchar#!#nvarchar#!#nvarchar
+master#!#dbo#!#xp_instance_regread#!#master#!#dbo#!#xp_instance_regread#!#<NULL>
```

Comment:
Is this expected to display "xp_instance_regread"?
I only see this in master database with rds sql server
```
SPECIFIC_NAME                                                                                                                   
--------------------------------------------------------------------------------------------------------------------------------
rds_is_db_read_replica                                                                                                          
rds_is_db_writable                                                                                                              
rds_hexadecimal                                                                                                                 
rds_help_revlogin                                                                                                               
rds_startup_tasks                                                                                                               
sp_MSrepl_startup                                                                                                               
sp_MScleanupmergepublisher 
```

Do we even want to display any proc/function created in pltsql dialect?

Reply:
No, we don't want to display these `xp_*` procedures. Working on it.